Src/libCZI/CziParse.cpp

```diff
@@ -573,6 +573,22 @@ using namespace libCZI;
         {
             libCZI::DimensionIndex dim = CCZIParse::DimensionCharToDimensionIndex(subBlkDirDV->DimensionEntries[i].Dimension, 4);
             entry.coordinate.Set(dim, subBlkDirDV->DimensionEntries[i].Start);
+
+            // check all the dimensions here. cr mark
+            if(options.GetStoredDimensionOtherThanMMustHaveSizeOne())
+            { 
+                auto storedSize = subBlkDirDV->DimensionEntries[i].StoredSize;
+                if(storedSize != 1)
```

Comment:
I wonder - can't we (and shouldn't we better) check both Size and StoredSize for being "1" here?

Reply:
Ehm no - or that's what is alreay checked with the existing option GetDimensionOtherThanMMustHaveSizeOne.

Proposal - we rename the option to "PhysicalDimensionMustBeSameAsLogical" and check whether physical extent is equal to logical.

Reply:
This is checked some lines below with the `options.GetDimensionOtherThanMMustHaveSizeOne()` flag.
So I thought the logic is clearer if we check seperately.

Reply:
Ja, makes sense, or I reckon the nomenclature you arrived at is as good as what I was suggesting. But please consider using the term "physical size" in external API, no need to add more confusion by making this internal and badly-chosen-name "stored size" more prominent.